File: tailo_realm_parser_app/src/main/java/com/taccotap/taigidictparser/MainActivity.java
package com.taccotap.taigidictparser;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.taccotap.taigidictparser.tailo.parser.TlParseIntentService;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TlParseIntentService.startParsing(this);
        finish();
    }
}

File: tailo_realm_parser_app/src/main/java/com/taccotap/taigidictparser/MainActivity.kt
package com.taccotap.taigidictparser

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.taccotap.taigidictparser.tailo.parser.input.KipParseIntentService

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        KipParseIntentService.startParsing(this)
        finish()
    }
}
